pkg/controller/account/byoc.go

```diff
@@ -85,12 +88,26 @@ func (r *ReconcileAccount) initializeNewCCSAccount(reqLogger logr.Logger, accoun
 
 	accountID := account.Labels[fmt.Sprintf("%s", awsv1alpha1.IAMUserIDLabel)]
 
+	// Get SRE Access ARN from configmap
+	cm := &corev1.ConfigMap{}
+	cmErr := r.Client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if cmErr != nil {
+		reqLogger.Error(cmErr, "There was an error getting the ConfigMap to get the SRE Access Role")
+		return "", reconcile.Result{}, cmErr
+	}
+
+	SREAccessARN := cm.Data["CCS-Access-Arn"]
+	if SREAccessARN == "" {
+		reqLogger.Error(fmt.Errorf("CCS-Access-Arn Key missing from ConfigMap %s/%s", awsv1alpha1.AccountCrNamespace, awsv1alpha1.DefaultConfigMap), "The key is missing from the ConfigMap")
```

Comment:
We have an ["awsv1alpha1.ErrInvalidConfigMap](https://github.com/openshift/aws-account-operator/blob/bbda1c91242b7d0cd78456e3cda55201dc9188f0/pkg/apis/aws/v1alpha1/shared_types.go#L97-L98)" type, and I would argue missing an expected/required value makes the configMap invalid.  Would it be worth using here?

```suggestion
		reqLogger.Error(awsv1alpha1.InvalidConfigMap, "configmap key missing", "keyName", "CCS-Access-Arn")
```
The `"keyName", "CCS-Access-Arn"` will append a field to the logline so that it's indexable/searchable in a log aggregator.

The ReqLogger already [includes the NameSpace as one of these extra appended fields](https://github.com/openshift/aws-account-operator/blob/bbda1c91242b7d0cd78456e3cda55201dc9188f0/pkg/controller/account/account_controller.go#L136), so that can be omitted.

Including the configMap name is fine, or not.  I don't have any opinion on that.